src/feature/BoltFeature.ts

```diff
@@ -0,0 +1,66 @@
+import { ExtensionContext, commands, window, Uri } from 'vscode';
+import { IFeature } from '../feature';
+import * as path from 'path';
+import * as fs from 'fs';
+
+export class BoltConfigFeature implements IFeature {
+  dispose() {}
+  constructor(context: ExtensionContext) {
+    context.subscriptions.push(
+      commands.registerCommand('puppet-bolt.OpenUserConfigFile', () => {
+        let userInventoryFile = path.join(process.env['USERPROFILE'] || '~', '.puppetlabs', 'bolt', 'bolt.yml');
+
+        if (!fs.existsSync(userInventoryFile)) {
+          window
+            .showQuickPick(['yes', 'no'], {
+              placeHolder: `Default bolt config yml not present. Do you want to create it?`,
+              canPickMany: false,
+              ignoreFocusOut: true
+            })
+            .then(answer => {
+              switch (answer) {
+                case 'no':
+                  break;
+                case 'yes':
+                  fs.writeFile(userInventoryFile, '', function(err) {});
+                  commands.executeCommand('vscode.openFolder', Uri.file(userInventoryFile), false);
+              }
+            });
+        } else {
+          commands.executeCommand('vscode.openFolder', Uri.file(userInventoryFile), false);
+        }
+      })
+    );
+  }
+}
+
+export class BoltInventoryFeature implements IFeature {
+  dispose() {}
+  constructor(context: ExtensionContext) {
+    context.subscriptions.push(
+      commands.registerCommand('puppet-bolt.OpenUserInventoryFile', () => {
+        let userInventoryFile = path.join(process.env['USERPROFILE'] || '~', '.puppetlabs', 'bolt', 'inventory.yml');
```

Comment:
Docs and code say the file is called `inventory.yaml`

https://github.com/puppetlabs/bolt/blob/b2a3511bf04f7cf9b42eeddff215b6ddcf75765b/lib/bolt/boltdir.rb#L35

https://puppet.com/docs/bolt/1.x/inventory_file.html

Reply:
👍